analyzer/methods/method_utils.py

```diff
@@ -3,6 +3,38 @@
 import pandas as pd
 import pygeos
 
+from analyzer import Geometry, TrajectoryData
+
+
+def validate_trajectory(traj: TrajectoryData, geometry: Geometry) -> bool:
```

Comment:
Since this function does not *validate* the trajectory, but merely tells is a trajectory is valid or not, I would rename it to  `is_valid()` or `is_trajectory_valid()` or something similar.